torch/quantization/fx/convert.py

```diff
@@ -45,6 +45,8 @@
     activation_dtype,
 )
 
+from .lower_to_fbgemm import lower_to_fbgemm
```

Comment:
The convert function here (L145) is almost 400 lines long. Can you refactor it in a later PR? With multiple functions defined inside, it is hard to understand the flow here.

Reply:
yes sure, will do

Reply:
I think we also might be able to simplify a bit after we change the flow to produce reference pattern by default